room/integration-tests/testapp/src/androidTest/java/androidx/room/integration/testapp/test/DatabaseCallbackTest.java

```diff
@@ -205,6 +211,82 @@ public void onDestructiveMigration_calledOnUpgrade() {
         database.close();
     }
 
+    @Test
+    public void onCreateFromAsset_calledOnOpenPrepackagedDatabase() {
```

Comment:
Can you plz move these test to https://github.com/androidx/androidx/blob/androidx-master-dev/room/integration-tests/testapp/src/androidTest/java/androidx/room/integration/testapp/test/PrepackageTest.java? These tests are for the old callbacks and since we are scoping the new one I think it might be best to have it with the rest of prepackage DB tests.